app/routes/admin/delete/owners.js

```diff
@@ -46,56 +44,36 @@ module.exports = [
       auth: { scope: [admin] },
       validate: {
         payload: orphanedOwnersPayloadSchema,
-        failAction: async (_, h, errors) => {
-          console.error(errors)
+        failAction: async (request, h) => {
```

Comment:
Any reason why you have the sort redirect in the failAction as opposed to it being a valid option in the schema (and therefore in the handler)?

Reply:
I guess it was to avoid having too much in the handler, but I see your point.  I can try moving.  Hopefully won't take too much time.